packages/menu/src/Menu/Menu.stories.jsx

```diff
@@ -193,6 +193,16 @@ export const UsingButtonPropsPropForStylingButtonText = () => (
   </Menu>
 )
 
+export const InDialog = () => (
+  <Dialog open size='md'>
+    <Box height={200} p={4}>
+      <Menu width={300} buttonText='Menu'>
+        <MenuItems />
+      </Menu>
+    </Box>
+  </Dialog>
```

Comment:
@JordonHong Could you add an interaction test that clicks to open the menu so Chromatic includes that in the snapshot?

Reply:
@craigpalermo sorry for the wait, just added in the interaction test. Lmk if it looks good!